com.ibm.streamsx.json/com.ibm.streamsx.json/types.spl

```diff
@@ -14,3 +14,28 @@ namespace com.ibm.streamsx.json;
 * [JSONToTuple] and [TupleToJSON].
 */
 type Json = rstring jsonString;
+
+
+public composite JsonIndex {
```

Comment:
@ddebrunner JsonIndex holds reference id enums to use with parseJSON and queryJSON.
Because parseJSON parses JSON to internal object you cannot return in to SPL, so you must give it some reference: `parseJSON(jsonStr, JsonIndex._1)`
then you can refer it later with queryJSON: `queryJSON("/root/t2", false, JsonIndex._1)`